Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/CleverTap/clevertap-ios-sdk.git", .upToNextMajor(from: "5.0.0")),
+        .package(url: "https://github.com/CleverTap/clevertap-ios-sdk.git", .upToNextMajor(from: "6.0.0")),
```

Comment:
@kushCT Should this be 6.1.0?

Reply:
@darshanclevertap I am following the legacy code architecture. Package.swift was set as 5.0.0 when 5.2.0 was added to the geofence sdk. But yeah I need to set it to minimum as 6.1.0 for the mandatory nsprivacy support. I am updating it to 6.1.0